src/network.c

```diff
@@ -1683,6 +1684,36 @@ static int network_set_interface(const sockent_t *se,
   return 0;
 } /* }}} network_set_interface */
 
+static int network_bind_socket_to_addr(sockent_t *se, const struct addrinfo *ai) {
+
+  if (se->bind_address == NULL)
+    return 0;
+
+  DEBUG("fd %i: bind socket to address", se->data.client.fd);
+  char pbuffer[64];
+
+  if (ai->ai_family == AF_INET) {
+    struct sockaddr_in *addr = (struct sockaddr_in *)(se->bind_address);
+    inet_ntop(AF_INET, &(addr->sin_addr), pbuffer, 64);
+    INFO("binding client socket to ipv4 address: %s", pbuffer);
+    if (bind(se->data.client.fd, (struct sockaddr*)addr, sizeof(*addr)) == -1) {
+      ERROR("network_bind_socket_to_addr: %s", STRERRNO);
+      return -1;
+    }
+  } else if (ai->ai_family == AF_INET6) {
+    struct sockaddr_in6 *addr = (struct sockaddr_in6 *)(se->bind_address);
+    inet_ntop(AF_INET, &(addr->sin6_addr), pbuffer, 64);
+    INFO("binding client socket to ipv6 address: %s", pbuffer);
+    if (bind(se->data.client.fd, (struct sockaddr*)addr, sizeof(*addr)) == -1) {
+      ERROR("network_bind_socket_to_addr: %s", STRERRNO);
```

Comment:
I think `INFO()` messages are excessive. 
And `ERROR()` messages should have prefix `network plugin: ` and be more descriptive.
Common style messages are look like this: `network plugin: Failed to bind client socket (ipvX): %s`, STRERRNO`.



See `network_set_ttl(), network_set_interface()` as reference.